src/pages/[lang]/projects/[project]/accept-invite.astro

```diff
@@ -0,0 +1,44 @@
+---
+import HeaderLayout from '@layouts/HeaderLayout.astro';
+import { createSupabaseServerClient } from '@backend/supabaseServerClient';
+import { getMyProfile } from '@backend/crud';
+import { getLangFromUrl, getTranslations } from '@i18n';
+import { AcceptInvite as AcceptInviteApp } from '@apps/accept-invite';
+
+const lang = getLangFromUrl(Astro.url);
+
+const i18n = getTranslations(Astro.request, 'dashboard-projects');
+
+const supabase = await createSupabaseServerClient(Astro.cookies);
+
+const projectId = Astro.params.project;
+
+const profile = await getMyProfile(supabase);
+if (profile.error || !profile.data) {
+  return Astro.redirect(`/${lang}/sign-in?redirect-to=${Astro.url}`);
+}
+
+console.log('Here!');
+const invitation = await supabase
```

Comment:
This could potentially return undefined, right? The app component expects an invitation though. Probably check for 

```if (!invitation.data || invitation.error)...```

below?